provision/Makefile

```diff
@@ -11,7 +11,7 @@ dist: test
 
 test:
 	flake8 --ignore E501,E731,E741,W504 acc_provision
-	python -m pytest acc_provision
+	python -m pytest -vv acc_provision
```

Comment:
you probably used this for your debugging, can you please remove it?

Reply:
done